Sources/MongoSwift/MongoCollection+Read.swift

```diff
@@ -71,22 +71,8 @@ extension MongoCollection {
         }
     }
 
-    // TODO SWIFT-133: mark this method deprecated https://jira.mongodb.org/browse/SWIFT-133
-    /**
-     * Counts the number of documents in this collection matching the provided filter.
-     *
-     * - Parameters:
-     *   - filter: a `Document`, the filter that documents must match in order to be counted
-     *   - options: Optional `CountOptions` to use when executing the command
-     *   - session: Optional `ClientSession` to use when executing this command
-     *
-     * - Returns: The count of the documents that matched the filter
-     *
-     * - Throws:
-     *   - `ServerError.commandError` if an error occurs that prevents the command from performing the write.
-     *   - `UserError.invalidArgumentError` if the options passed in form an invalid combination.
-     *   - `EncodingError` if an error occurs while encoding the options to BSON.
-     */
+     /// :nodoc:
+    @available(*, deprecated, message: "Use countDocuments or estimatedDucmentCount instead.")
```

Comment:
We plan on removing this for 1.0 right?

Reply:
Yes, should I just remove the original `count` function altogether?

Reply:
We're going to be breaking tons of API next release, so might as well just remove this altogether as part of that.